backend/de.metas.banking/de.metas.banking.base/src/main/sql/postgresql/system/5690470_sys_gh15490_t_getopenpayments_paymentdate_type_fix.sql

```diff
@@ -0,0 +1,85 @@
+DROP FUNCTION IF EXISTS getopenpayments(numeric,
+                                        numeric,
+                                        timestamp WITHOUT TIME ZONE,
+                                        numeric)
+;
+
```

Comment:
keep the previous drop, but also pls drop the current version of the function, i.e.
```sql
DROP FUNCTION IF EXISTS getopenpayments(numeric,
                                        numeric,
                                        timestamp WITH TIME ZONE,
                                        numeric)
;
```